docs/Actions/send-email.md

```diff
@@ -0,0 +1,531 @@
+| Subject                       | The subject of the Email                                                                                                                                                                                                                                                  | Yes           | `Field is required`                      |
+| Format                        | The format of the email body                                                                                                                                                                                                                                              | No              | `Html (And replace newline with <br />)` |
+| DNN email template to use.    | Choose from a predefined email from the system                                                                                                                                                                                                                            | No              | Unset                                    |
+| Body                          | The email template you want to send                                                                                                                                                                                                                                       | Yes             | `empty string`                           |
+| Headers                       | Append custom headers to the email.                                                                                                                                                                                                                                       | Yes             | Unset                                    |
+| Attach Portal Files           | Pick a file that you want to send as attachment via email                                                                                                                                                                                                                 | No              | Unset                                    |
+| Attach Field #                | Form only parameters, provide the ability to add up to 5 Single File Uploads fields to be send as attachments to the email                                                                                                                                                | No              | Unset                                    |
+| Attach from Token             | This lets you compute file path from token. It must be relative to portal home folder. Multiple items must be separated by semicolons.                                                                                                                                    | Yes             | `empty string`                           |

```

Comment:
```suggestion
| Attach from Token             | Computes file path from token. It must be relative to the portal home folder. Multiple items must be separated by semicolons.                                                                                                                                    | Yes             | `empty string`                           |

```

Reply:
Do they have to be relative to the portal folder?
I remember being able to attach PDF files using the absolute URL or even the full file path.